src/mame/irem/olibochu.cpp

```diff
@@ -298,41 +299,48 @@ u32 olibochu_state::screen_update(screen_device &screen, bitmap_ind16 &bitmap, c
 
 TIMER_DEVICE_CALLBACK_MEMBER(olibochu_state::scanline)
 {
-	int scanline = param;
+	// see TODO
+	int const scanline = param;
 
-	if (scanline == 248) // vblank irq
-		m_maincpu->set_input_line_and_vector(0, HOLD_LINE, 0xd7); // Z80 - RST 10h
+	if (scanline == 248) // VBLANK
```

Comment:
Are you sure that sound and main IRQs are triggered at the same time? Does it introduce a chance of main-sound comms going wrong?